src/index.js

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './components/App';
+// import App from './components/App';
 import * as serviceWorker from './serviceWorker';
 import store from './redux/store';
 import { saveState } from './helpers/localStorage';
+import Routes from './Router';
+
 
 // update localstorage whenever state changes
 store.subscribe(() => {
     saveState(store.getState());
```

Comment:
indenting

Reply:
Could you kindly elaborate on what you mean with this comment?

Reply:
@Mubangizi this has been fixed.